interfaces/builtin/raw_usb.go

```diff
@@ -51,6 +51,14 @@ const rawusbConnectedPlugAppArmor = `
 /run/udev/data/b180:*    r, # various USB block devices
 /run/udev/data/c18[089]:* r, # various USB character devices: USB serial converters, etc.
 /run/udev/data/+usb:* r,
+
+# Allow binding/unbind of usb driver from usb devices
```

Comment:
The raw-usb interface would appear to be about providing access to the underlying USB devices - but should that also include the ability to control what driver is bound to each device? Would this be more appropriate in a `usb-control` interface instead?

Reply:
@alexmurray Apologies for the delayed response.

I can see why you would suggest that. To be honest, I was just trying to make a quick contribution to enable the use of USB/IP on Ubuntu Core, and was hoping for a quick win.

Right now it can work using the `system-files` interface, but that's not ideal for a few reasons, the main one being that you can't use wildcards and so pci paths have to be hard coded which is obviously not a sustainable approach.

If creating a new interface is the only way forward, I will probably not be able to complete this, as I'm lacking some background in go along with the time to dedicate to it.

Please let me know your thoughts.